lms/templates/registration/account_activation_sidebar_notice.html

```diff
@@ -0,0 +1,34 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="profile-sidebar" role="region" aria-label="Account Activation Info">
+  <header class="profile">
+    <h2 class="account-activation-title sr">${_("Account Activation Info")}: </h2>
+  </header>
+  <div class="user-info">
+    <ul>
+      <li class="status status-verification warning" role="alert" aria-labelledby="title status-title" tabindex="1">
+        <span class="title status-title">
+          <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
```

Comment:
This icon is not accessible.  There should be SR text that conveys the same meaning as the visual icon ```<span class="sr">Notice</span>```